docs/sources/next/examples/get-started-with-k6/reuse-and-re-run-tests.md

```diff
@@ -136,121 +136,125 @@ As is, this script won't work, since it has undeclared functions and variables.
 
 1. Finally, pass `baseUrl` as a parameter of the `contacts` function.
 
-  {{< code >}}
-  ```javascript
-  import http from 'k6/http';
```

Comment:
@jdbaldry I just noticed that with our current configuration of prettier, we might lose the way we add spacing to text between numbered list items...

Do you know if there's an easy way to fix this?

Reply:
`prettier` removes most of the ambiguity in Markdown parsing but in doing so, it can impose some formatting that seems different to our mental models of Markdown parsing.

Is this section is rendering differently than it was before running `prettier` on this file? 

With regards to list formatting, I think it is useful to read https://spec.commonmark.org/0.30/#list-marker and see if that helps.

Reply:
Sorry, I should've marked the whole section 😅 .

But yeah, this is removing the indentation between the numbered items, so it's affecting how it renders it, and also making the list display `1.`, and `1.` for the next list item, instead of `1.` and `2.`.

Reply:
I think the indentation that was there before `prettier` formatting had ambiguous parsing. The way `prettier` formatted it indicates to me that `prettier` didn't parse those sections as lazy numbered ordered list blocks.

I've opened https://github.com/grafana/k6-docs/pull/1461 which shows the `prettier` preferred indentation for list blocks that preserves lazy numbering. Hopefully this helps :)

Reply:
Ooh you're right! The file had 2 spaces instead of 3. Sorry, I should've looked closer at it. 😓 

Thanks for your help, I'll try to go through the files to make sure all the lists are formatted correctly before I merge this. 🙏

Reply:
No worries at all, I'm glad you spotted it. Indentation is a real pain. Fortunately, once `prettier` is in place, things should be easier :)